tests/std/tests/P2674R1_is_implicit_lifetime/test.compile.pass.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <type_traits>
+
+enum UnscopedEnum { Enumerator };
+enum struct ScopedEnum { Enumerator };
+class IncompleteClass;
+struct TrivialClass {};
+struct AggregateClassWithUserProvidedDestructor {
+    ~AggregateClassWithUserProvidedDestructor() {}

```

Comment:
I'd like to use a shorter name.
```suggestion
struct UserProvidedDestructorClass {
    ~UserProvidedDestructorClass() {}
```